supabase/migrations/20260723144749_dedicated_native_builder.sql

```
@@ -0,0 +1,139 @@
+ALTER TABLE "public"."build_requests"
+  ADD COLUMN IF NOT EXISTS "builder_pool" text;
+
+COMMENT ON COLUMN "public"."build_requests"."builder_pool" IS
+  'Pool preferred/used for this job: dedicated or shared. Null for legacy rows.';
+
+ALTER TABLE "public"."build_requests"
+  DROP CONSTRAINT IF EXISTS "build_requests_builder_pool_check";
+
+ALTER TABLE "public"."build_requests"
+  ADD CONSTRAINT "build_requests_builder_pool_check"
+  CHECK (("builder_pool" IS NULL OR "builder_pool" = ANY (ARRAY['dedicated'::text, 'shared'::text])));
```

SQLFluff (4.2.2)

[error] Line is too long (102 > 80).

(LT05)